package.json

```diff
@@ -99,9 +99,14 @@
     "lodash": "4.11.1",
     "raven-js": "3.0.4",
     "react": "15.1.0",
+    "react-addons-shallow-compare": "15.1.0",
+    "react-addons-test-utils": "15.1.0",
     "react-autosuggest": "3.7.4",
     "react-bootstrap": "0.29.4",
+    "react-color": "^2.2.0",
     "react-dom": "15.1.0",
+    "react-draggable": "^2.1.2",
+    "three": "^0.78.0",
```

Comment:
For `react-color`, `react-draggable`, and `three`, use exact version numbers (remove the carat `^`)